src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -208,6 +209,8 @@ public void deleteTeam(Island island, User user) {
         IridiumSkyblock.getInstance().getDatabaseManager().getIslandTableManager().delete(island);
 
         getMembersOnIsland(island).forEach(member -> PlayerUtils.teleportSpawn(member.getPlayer()));
+
+        Bukkit.getPluginManager().callEvent(new IslandDeleteEvent(island, user));
```

Comment:
Should check the return value and not call the delete methods if the event is cancelled (hence calling it before like 209 with a if statement about the cancelled state